nabo/nabo.h

```diff
@@ -217,11 +217,13 @@ namespace Nabo
 	// TODO (c++14) Convert invalidIndex, invalidValue to constexpr templated variables.
 	template <typename IndexType>
 	inline constexpr IndexType invalidIndex() {
+		static_assert(std::numeric_limits<IndexType>::is_integer, "");
```

Comment:
Minor: I'd suggest `std::is_integral` instead. That is a bit more symmetric and numeric_limits is not defined for non numeric types. The error might differ for some compilers between a false value and a substitution failure. I don't know what the standard says here. Does somebody else?

Reply:
From the C++11 standard section 18.3.2.3: *The default numeric_limits<T> template shall have all members, but with 0 or false values.* Still, I will change it to `std::is_integral`.

Reply:
Ah, thanks! I believed otherwise. Thanks for changing anyways.